### PR TITLE
[AppBar] Create an example of MDCAppBarContainerViewController usage

### PR DESCRIPTION
**Pull request by jmdetloff**
Closes: https://github.com/material-components/material-components-ios/issues/3807

## Screenshots

simulator screen shot - iphone 8 plus - 2018-05-18 at 13 37 48

### CODE REVIEW

File: components/AppBar/examples/AppBarWrappedExample.m [hunk 1]
```diff
@@ -0,0 +1,97 @@
+/*
+ Copyright 2016-present the Material Components for iOS authors. All Rights Reserved.
```
Comment (jverkoey) [inline components/AppBar/examples/AppBarWrappedExample.m]: Copyright.

**Review by jverkoey — APPROVED** [review]

File: components/AppBar/examples/AppBarWrappedExample.m (hunk 1 above)
Comment (jmdetloff) [reply components/AppBar/examples/AppBarWrappedExample.m]: Updated

**Comment by jverkoey** [comment]
Is this ok to merge?

**Comment by jverkoey** [comment]
Is this ready to merge?

**Pull request merged**